### PR TITLE
RAINCATCH-1030 Remove legacy code from demo-portal and demo-mobile

### PR DESCRIPTION
**Pull request by wtrocki**
## Motivation

Remove legacy code from demo-portal and demo-mobile

## Progress
- [x] Cleanup 
- [x] Remove legacy code from demo-portal
- [x] Remove legacy code from demo-mobile
- [x] Enable travis for repository
- [x] Documentation fixes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,23 @@
-# RainCatcher
-Feedhenry RainCatcher Angularjs Repository
-Default UI implementation for Raincatcher
+# Feedhenry RainCatcher Angularjs
 
-## About RainCatcher angularjs
+Reference mobile and website implementation for RainCatcher
 
-Raincatcher angularjs is default UI implementation for [Raincatcher Core Framework](https://github.com/feedhenry-raincatcher/raincatcher-core).
+## About RainCatcher ~ngularjs
```
Comment (paolobueno) [inline README.md]: little typo here

File: demo/mobile/config.xml [hunk 2]
```diff
@@ -2,9 +2,9 @@
 <widget id="com.redhat.demo.wfm" version="0.0.1" xmlns="http://www.w3.org/ns/widgets" xmlns:cdv="http://cordova.apache.org/ns/1.0">
   <name>FeedHenry WFM</name>
   <description>
-        A demo of the RHMAP WFM modules.
+      A demo mobile of the Feedhenry Raincatcher
```
Comment (paolobueno) [inline demo/mobile/config.xml]: I think the capitalization is RainCatcher? we always confuse it :D

File: demo/mobile/package.json [hunk 3]
```diff
@@ -23,22 +23,8 @@
...
-    "fh-wfm-sync": "1.0.0",
-    "fh-wfm-user": "0.6.6",
-    "fh-wfm-user-angular": "0.2.5",
-    "fh-wfm-vehicle-inspection": "0.0.15",
-    "fh-wfm-workflow": "0.2.6",
-    "fh-wfm-workflow-angular": "0.2.6",
-    "fh-wfm-workorder": "0.2.4",
-    "fh-wfm-workorder-angular": "0.3.3",
+    "@raincatcher/workflow-angular": "1.0.0",
+    "@raincatcher/workorder-angular": "1.0.0",
```
Comment (paolobueno) [inline demo/mobile/package.json:27]: :+1: for new npm modules

File: demo/mobile/src/app/config.js [hunk 4]
```diff
@@ -11,21 +11,17 @@ function createMainAppRoute($stateProvider, $urlRouterProvider) {
...
-        }
+        // profileData: function(userClient) {
+        //   return userClient.getProfile();
+        // },
+        // workorderManager: function(syncManagers) {
+        //   return syncManagers.workorders;
+        // },
+        // workflowManager: function(syncManagers) {
+        //   return syncManagers.workflows;
+        // }
```
Comment (paolobueno) [inline demo/mobile/src/app/config.js:22]: keeping this commented out code to remove in another PR? it's fine by me, but we gotta remember to take it out

**Review by paolobueno — APPROVED** [review]
LGTM, minor typos and such.

Now I understood that the commented out code is for when the other modules are okay.

File: demo/mobile/src/app/config.js (hunk 4 above)
Comment (wtrocki) [reply demo/mobile/src/app/config.js:22]: Yeah. I'm having changes as part of another ticket and PR and wanted this to be merged in order to bring that in.

**Pull request merged**